lib/Ray/sources/Drawables/2D/Triangle.cpp

```diff
@@ -52,9 +52,8 @@ void RAY::Drawables::Drawables2D::Triangle::drawOn(RAY::Window &)
 	DrawTriangle(this->_position, this->_posB, this->_posC, this->_color);
 }
 
-void RAY::Drawables::Drawables2D::Triangle::drawOn(RAY::Image &image)
+void RAY::Drawables::Drawables2D::Triangle::drawOn(RAY::Image &)
 {
-	(void)image;
 	throw std::exception();
```

Comment:
You should throw a specific exception. That means a derived of std::runtime_exception with a specifc type and a specfic error message.

Reply:
Here that could be a NotSupportedError with a message of "A triangle can't be drawn to an image".